### DISCUSSION TITLE
What is the default TextBlock foreground color resource?

### DISCUSSION THREAD

**Discussion by alexandrehtrb** [AvaloniaUI/Avalonia#11510] [Q&A] [answered]
Hello,

I am trying to make a multi-theme app and I would like to change the default TextBlock text color (foreground).

I checked both the Avalonia repo and the [Avalonia.ThemeManager](https://github.com/wieslawsoltes/Avalonia.ThemeManager) repo and couldn't discover what is the default color resource.

Which is it and where it is defined?

**Comment by maxkatz6** [comment, marked as answer]
Foreground is set on the top level control (Window class usually), and inherited down to all controls.
For the fluent theme you can find it here https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/Window.xaml#L7
